music21/converter/subConverters.py

```diff
@@ -732,12 +732,10 @@ def findPNGfpFromXMLfp(self, xmlFilePath):
         Check whether total number of pngs is in 1-9, 10-99, or 100-999 range,
          then return appropriate fp. Raises and exception if png fp does not exist.
         '''
-        if os.path.exists(xmlFilePath[0:len(xmlFilePath) - 4] + "-1.png"):
-            pngfp = xmlFilePath[0:len(xmlFilePath) - 4] + "-1.png"
-        elif os.path.exists(xmlFilePath[0:len(xmlFilePath) - 4] + "-01.png"):
-            pngfp = xmlFilePath[0:len(xmlFilePath) - 4] + "-01.png"
-        elif os.path.exists(xmlFilePath[0:len(xmlFilePath) - 4] + "-001.png"):
-            pngfp = xmlFilePath[0:len(xmlFilePath) - 4] + "-001.png"
+        import glob
+        found = sorted(glob.glob(xmlFilePath[0:len(xmlFilePath) - 4] + "-*.png"))
+        if found and len(found) < 999:
```

Comment:
we proibably don't need to check for < 999 now, since your glob will catch even longer generations.